pytorch_lightning/trainer/seed.py

```diff
@@ -0,0 +1,29 @@
+"""Helper functions to help with reproducibility of models """
+import os
+
+import numpy as np
+import random
+import torch
+
+from pytorch_lightning import _logger as log
+
+
+def seed_everything(seed: int = None):
```

Comment:
Why is this implemented twice (here and in utilities/reproducibility)? Probably a rebasing issue? I'd dismiss this one and apply the changes (e.g. removal of the explicit cuda seeds and the cudnn flag) to the utilities version

Reply:
Should be a rebasing issue. I moved code from initial `reproducibility.py` to `seed.py`. I will delete it now